CircleciScripts/bump_sdk_version.py

```diff
@@ -0,0 +1,137 @@
+from functions import replacefiles
+import sys
+import os
+import glob
+from lxml import etree
+def bump_plist(filename, newsdkversion):
+    tree = etree.parse(filename)
+    root = tree.getroot()
+    namespaces = root.nsmap
+    dictnode = root.find("./dict", namespaces)
+    setversion = False
+    for child in dictnode:
+        if child.tag == 'key' and child.text == 'CFBundleShortVersionString':
+            setversion = True
+        else:
+            if setversion:
+                print("old version:", child.text)
+                child.text=newsdkversion
+                break
+    tree.write(filename)
+
+
+root = sys.argv[1]
+newsdkversion = sys.argv[2]
+modulelist = [
+    "AWSAPIGateway",
+    "AWSAutoScaling",
+    "AWSCloudWatch",
+    "AWSCognito",
+    "AWSCognitoAuth",
+    "AWSCognitoIdentityProvider",
+    "AWSComprehend",
+    "AWSCore",
+    "AWSDynamoDB",
+    "AWSEC2",
+    "AWSElasticLoadBalancing",
+    "AWSIoT",
+    "AWSKMS",
+    "AWSKinesis",
+    "AWSKinesisVideo",
+    "AWSKinesisVideoArchivedMedia",
+    "AWSLambda",
+    "AWSLex",
+    "AWSLogs",
+    "AWSMachineLearning",
+    "AWSMobileAnalytics",
+    "AWSPinpoint",
+    "AWSPolly",
+    "AWSRekognition",
+    "AWSS3",
+    "AWSSES",
+    "AWSSNS",
+    "AWSSQS",
+    "AWSSimpleDB",
+    "AWSTranscribe",
+    "AWSTranslate",
+    'AWSAutoScaling',
+    'AWSAuthSDK/Sources/AWSAuthCore',
```

Comment:
I . think we should also have `AWSAuthUI` here.

Reply:
done